spark/interpreter/src/test/java/org/apache/zeppelin/spark/SparkShinyInterpreterTest.java

```diff
@@ -102,7 +102,7 @@ public void testSparkShinyApp() throws IOException, InterpreterException, Interr
     Thread.sleep(5 * 1000);
     // extract shiny url
     List<InterpreterResultMessage> resultMessages = context2.out.toInterpreterResultMessage();
-    assertEquals(1, resultMessages.size());
+    assertEquals(resultMessages + "", 1, resultMessages.size());
```

Comment:
The result seems be the same as `resultMessages.toString()`, but wouldn't it be better to use a consistent expression with other code?

Reply:
This change is only for diagnose to display the content of resultMessages when this assertion fails.

Reply:
> The result seems be the same as `resultMessages.toString()`, but wouldn't it be better to use a consistent expression with other code?

Right, let me update it